doc_user/UserManual.adoc

```diff
@@ -2625,14 +2630,14 @@ To insert a new ward, provide the following required inputs:
 * *Number of nurses*: it may be shown in some reports (0 – zero is allowed)
 * *Number of doctors*: it may be shown in some reports (0 – zero is allowed)
 * *Visit duration*: length of a visit (in minutes and must be greater than zero)
-* *Ward with pharmacy*: it is used in Pharmaceuticals Stock; if checked it means that ward can be a destination for discharging movements, otherwise it will be not shown in the list
+* *Ward with OPD*: if checked, it means that the ward will also be available in the <<opd-extended,OPD Extended>> mode
+* *Ward with pharmacy*: it is used in Pharmaceuticals Stock; if checked, it means that the ward can be a destination for discharging movements, otherwise it will be not shown in the list
 * *Male Ward*: if it is a male Ward
 * *Female Ward*: if it is a female Ward
 
-NOTE: Using this function the definition of wards with 0 (zero) beds but with a pharmacy is possible.
-This allows for "logical" departments (rather than physical) to manage pharmacy movements.
+NOTE: Maternity (M) and OPD (OPD) wards are _built-in_ wards and cannot be deleted, although can be modified as preferred.
```

Comment:
`although can be modified as preferred.` -> `although they can be modified as preferred wards.`

Reply:
"as preferred" I mean "as one prefer"

Reply:
OK; then  `although they can be modified as required.` or `although they can be modified as needed.`